### PR TITLE
add zh-CN translation for hello world

### PR DESCRIPTION
**Pull request by runzexia**

### CODE REVIEW

**Comment by runzexia** [comment]
@jenkinsci/code-reviewers

**Comment by oleg-nenashev** [comment]
I am not 100% sure it is supposed to wok in such way. IIUC it should go to https://github.com/jenkinsci/localization-zh-cn-plugin , CC @LinuxSuRen

**Comment by runzexia** [comment]
@oleg-nenashev 
The hello world in this repository is prepared for plugin developers, so it should not be in localization-zh-cn-plugin

**Comment by LinuxSuRen** [comment]
In my opinions, perhaps there are two user cases: private or public plugin. If someone wants to create a public plugin (hosted in jenkinsci), then these localization files should be in ttps://github.com/jenkinsci/localization-zh-cn-plugin. Another case is the private plugin. They need to save these Chinese localization files in their plugin.
So my offer is that keep the Chinese localization files in the archetypes. Then add some comments to explain.

**Comment by LinuxSuRen** [comment]
@runzexia Do you still wish to make these changes?

**Comment by LinuxSuRen** [comment]
My suggestion is using the Chinese comment.

**Comment by runzexia** [comment]
add comments

**Comment by runzexia** [comment]
This PR should have no problem, can someone help merge?
@oleg-nenashev @daniel-beck

File: hello-world/src/main/resources/archetype-resources/src/main/resources/HelloWorldBuilder/config_zh_CN.properties [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# 如果您希望在 Jenkins 组织当中托管您的插件，那我们推荐您将本地化文件提交到 https://github.com/jenkinsci/localization-zh-cn-plugin 中。
+# 如果您希望私有维护插件，可以参考本例子的方式放置中文的本地化文件。
```
Comment (jglick) [inline hello-world/src/main/resources/archetype-resources/src/main/resources/HelloWorldBuilder/config_zh_CN.properties]: Google Translate FTR:

> If you wish to host your plugin in the Jenkins organization, we recommend that you submit your localization files to https://github.com/jenkinsci/localization-zh-cn-plugin.
> If you want a private maintenance plugin, you can refer to this example to place the Chinese localization file.

**Pull request merged**